### DISCUSSION TITLE
Should we bring ServiceModelEnumValidator - Corewcf.ConfigurationManager

### DISCUSSION THREAD

**Discussion by birojnayak** [CoreWCF/CoreWCF#448] [General]
Should we bring ServiceModelEnumValidator - Corewcf.ConfigurationManager

**Comment by mconnew** [comment]
ServiceModelEnumValidator uses internal helper methods to validate whether the value is valid. This was needed because methods on Enum were historically really slow. I believe they have sped up the implementation of the Enum methods in .NET Core so we could have a dynamic implementation which doesn't depend on a helper method.
